### DISCUSSION TITLE
Help trying to run locally (GraphQL)

### DISCUSSION THREAD

**Discussion by EdwinFajardoBarrera** [hoppscotch/hoppscotch#2014] [Q&A] [answered]
I'm trying to run the project locally (to contribute) but I got this error:

![image](https://user-images.githubusercontent.com/36389489/146307403-636ebf1e-a573-4ff3-b743-f4e538d81239.png)

In the browser I got this:

![image](https://user-images.githubusercontent.com/36389489/146307599-b8abdbb3-6431-4b17-8ef3-fb3ab8d81999.png)

Thanks for the help!

**Comment by liyasthomas** [comment]
@AndrewBastin

**Comment by AndrewBastin** [comment, marked as answer]
Make sure you have proper internet connectivity when and then try running `pnpm install` again

**Reply by EdwinFajardoBarrera** [reply]
Definitely was that. My proxy was blocking some connections. TSM!
